gulpfile.js

```diff
@@ -54,5 +55,22 @@ gulp.task('fonts', function() {
         .pipe(gulp.dest(path.join(output_directory, 'fonts')));
 });
 
+// Extract angular-gettext strings
+gulp.task('pot', function () {
+    return gulp.src(['openslides/core/static/templates/*.html',
+                     'openslides/users/static/templates/users/*.html',
+                     'openslides/users/static/js/users/*.js'])
+        .pipe(gettext.extract('template.pot', {}))
+        .pipe(gulp.dest('openslides/locale/angular-gettext/'));
+});
+// Compile angular-gettext strings
```

Comment:
done (Note: we don't compile mo-files. We convert the po files directly to json files which are saved in openslides/static/i18n and included by angular url loading.)